Sortable.js

```diff
@@ -736,9 +736,10 @@
 
 
 				if ((el.children.length === 0) || (el.children[0] === ghostEl) ||
-					(el === evt.target) && (_ghostIsLast(el, evt))
+					(el === evt.target) && (target = _ghostIsLast(el, evt))
```

Comment:
This assignment was removed in #1106 but the next `elseif` block depends on it. Since the assignment was removed that entire block was never executed which apparently caused the problems that we're experiencing.